coral/src/app/components/AuthenticationRequiredAlert.tsx

```diff
@@ -18,11 +18,11 @@ function AuthenticationRequiredAlert() {
         </span>
       </Typography.Heading>
 
-      <Typography.LargeText>
+      <Typography.DefaultStrong>
```

Comment:
This should also be `LargeStrong`, as is done here: https://github.com/Aiven-Open/klaw/pull/1854/files#diff-89a0b268afc943a78c3f355a3603ca7161aa814b2df7637bbded14b71f51e880R11